client/src/legacy/LeftAndMain.Panel.js

```diff
@@ -172,37 +183,18 @@ $.entwine('ss', function($) {
     }
   });
 
-  $('.cms-panel.collapsed .cms-panel-toggle').entwine({
```

Comment:
This is pretty pointless as the only thing this container has inside of it is the `<button>`, which has its own click handler